pkg/integration/tests/cherry_pick/cherry_pick_conflicts.go

```diff
@@ -53,6 +53,7 @@ var CherryPickConflicts = NewIntegrationTest(NewIntegrationTestArgs{
 			Confirm()
 
 		t.Common().AcknowledgeConflicts()
+		t.Views().Information().Content(Contains("2 commits copied"))
```

Comment:
Worth adding a comment explaining why we expect to see this i.e.:
"cherry pick selection is not cleared when there are conflicts, so that the user is able to abort and try again without having to re-copy the commits"

Reply:
Good point. Thank you for the feedback.

Reply:
@jesseduffield I amended  and force pushed the suggested comment.